features/full_tests/batch_2/native_crash_handling.feature

```diff
@@ -4,15 +4,22 @@ Scenario: Dereference a null pointer
         When I run "CXXNullPointerScenario" and relaunch the app
         And I configure the app to run in the "non-crashy" state
         And I configure Bugsnag for "CXXNullPointerScenario"
-        And I wait to receive a request
-        And the request payload contains a completed unhandled native report
+        And I wait to receive an error
+        And the error payload contains a completed unhandled native report
         And the exception "errorClass" equals "SIGSEGV"
         And the exception "message" equals "Segmentation violation (invalid memory reference)"
+# TODO: Or it might be:
```

Comment:
Is this comment still required?

Reply:
No, good spot - I've taken a note to remove on the very next branch, though, simply to avoid a whole new round of CI.